src/types/state.ts

```diff
@@ -142,9 +153,14 @@ const ServiceState = t.intersection([
 ]);
 export type ServiceState = t.TypeOf<typeof ServiceState>;
 
-const ReleaseState = t.type({
-	services: t.record(DockerName, ServiceState),
-});
+const ReleaseState = t.intersection([
+	t.type({
+		services: t.record(DockerName, ServiceState),
+	}),
+	t.partial({
+		update_status: UpdateStatus,
+	}),
```

Comment:
Why `t.partial`? Are there apps in this release that won't have `update_status`? I guess the host apps wouldn't, right?

Reply:
Actually I don't know. We don't report the status for host apps yet, but I assume it will work in a similar way. I'll probably make this required